### DISCUSSION TITLE
보물상자 개발 관련 질문

### DISCUSSION THREAD

**Discussion by pianoop** [SeoulTechTCPGame/HumanSurvival#247] [General]
[보물상자 이슈 링크](https://github.com/SUST22Unity/HumanSurvival/issues/238)  
구현에 대해 자세하게 적혀있지 않아 뱀서 게임대로 만드는 중입니다.
동전이 뿜어져 나오거나 다양한 장비들이 날아다니는 연출 등 세부 사항에 대한 항목들을 추가해주시면 감사하겠습니다!

**Comment by hgsanguk** [comment]
Issue에 애니메이션에 대한 구현 To-do와 예시 영상 링크를 추가했습니다.
